SQL/0000-00-00-schema.sql

```diff
@@ -246,6 +246,17 @@ CREATE TABLE `users` (
 INSERT INTO `users` (ID,UserID,Real_name,First_name,Last_name,Email,Privilege,PSCPI,DBAccess,Active,Pending_approval,Password_expiry)
 VALUES (1,'admin','Admin account','Admin','account','admin@example.com',0,'N','','Y','N','2016-03-30');
 
+CREATE TABLE `user_project_rel` (
+  `UserID` int(10) unsigned NOT NULL,
+  `ProjectID` int(2) NOT NULL,
+  PRIMARY KEY  (`UserID`,`ProjectID`),
+  KEY `FK_user_project_rel_2` (`ProjectID`),
+  CONSTRAINT `FK_user_project_rel_2` FOREIGN KEY (`ProjectID`) REFERENCES `Project` (`ProjectID`) ON DELETE CASCADE ON UPDATE CASCADE,
+  CONSTRAINT `FK_user_project_rel_1` FOREIGN KEY (`UserID`) REFERENCES `users` (`ID`) ON DELETE CASCADE ON UPDATE CASCADE
+) ENGINE=InnoDB DEFAULT CHARSET=utf8;
+
+INSERT INTO user_project_rel (UserID, ProjectID) SELECT 1, ProjectID FROM Project;
```

Comment:
Why does User 1 get all projects but no other users get any projects by default?

Reply:
https://github.com/aces/Loris/pull/4255/files#diff-49c955f1af7f1ad35cdd7a9aceb590d1R270
^ Related: Why does User 1 get all centers but no other users get any centers by default?

Possible answer: there's only User 1 upon sourcing the schema, and we can't know who are the additional users, and also this line will probably do nothing(at least so far) until we come up with a possible default project to insert...

Reply:
Ah.. I was thinking this was the patch for existing projects, not the default schema. It makes sense for the default schema.